.github/workflows/dockerhub-released-image.yml

```diff
@@ -45,8 +42,4 @@ jobs:
       - name: build and publish images
         env: 
           VERSION: ${{ github.ref_name }}
-          GOARCH: ${{ matrix.arch }}
-        # When we build binary for multiple platform, `go build` may be skipped as `$(SOURCES)` unchanged.
-        # So we clean the binary at first, and force to run `go build`.
-        # See https://github.com/karmada-io/karmada/pull/1649#issuecomment-1107959879 for more info.
-        run: make clean mp-image-${{ matrix.target }}
+        run: make mp-image-${{ matrix.target }}
```

Comment:
ditto.

Reply:
And how GitHub action checkout to the correct branch to release, but not on the master branch.

Reply:
> And how GitHub action checkout to the correct branch to release, but not on the master branch.

Sorry, I can't get it.